projects/plugins/jetpack/extensions/shared/components/media-player-control/index.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { ToolbarGroup, ToolbarButton } from '@wordpress/components';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { ControlBackFiveIcon, ControlForwardFiveIcon } from '../../icons';
+import { STATE_PAUSED, STORE_ID } from '../../../store/media-source/constants';
+import { convertSecondsToTimeCode } from './utils';
+
+function noop () {}
+
+export default function MediaPlayerControl( {
+	skipForwardTime = 5,
+	jumpBackTime = 5,
+	playIcon = 'controls-play',
+	pauseIcon = 'controls-pause',
+	onTimeChange = noop,
+} ) {
+	const {
+		playerState,
+		playerCurrentTime,
+		defaultMediaSource,
+	} = useSelect( select => {
+		const {
+			getMediaSourceCurrentTime,
+			getMediaPlayerState,
+			getDefaultMediaSource,
+		} = select( STORE_ID );
+
+		return {
+			playerState: getMediaPlayerState(),
+			playerCurrentTime: getMediaSourceCurrentTime(),
+			defaultMediaSource: getDefaultMediaSource(),
+		};
+	}, [] );
+
+	const timeInFormat = convertSecondsToTimeCode( playerCurrentTime );
+	const isDisabled = ! defaultMediaSource;
+
+	const { toggleMediaSource } = useDispatch( STORE_ID );
+	const togglePlayer = () => toggleMediaSource( defaultMediaSource.id );
+
+	return (
+		<ToolbarGroup>
+			{ jumpBackTime !== false && (
```

Comment:
It's really cool that we can add these skip and jump buttons to the controls but I wasn't able to get them working:
![skipbuttons](https://user-images.githubusercontent.com/1689238/104956001-3b9dd580-5999-11eb-8a1b-1e52f6b08902.gif)

Reply:
because it was not implemented yet :-D

Reply:
😆 That makes sense!